src/PointwiseFunctions/GeneralRelativity/ComputeGhQuantities.hpp

```diff
@@ -168,18 +170,19 @@ tnsr::i<DataType, SpatialDim, Frame> spatial_deriv_of_lapse(
  * \f$\Phi_{iab} = \partial_i \psi_{ab} \f$, and the operator
  * \f$D := \partial_0 - N^k \partial_k \f$. The time derivative of N is then:
  * \f{align*}
- *  (1/2) N^2 t^a t^b \Pi_{ab} - N N^i/2 t^a t^b \Phi_{iab}
- *  =& (1/2) N^2 t^a t^b t^c \partial_c \psi_{ab}
- *       - N N^i/2 (-(2/N) \partial_i N) \\
- *  =& (1/2) N^2 [-1/N^3 D[g_{jk} N^j N^k - N^2] - (N^j N^k / N^3)D[g_{jk}]
- *           + 2(N^j / N^3) D[g_{jk} N^k] + (2 / N^2)(N^i \partial_i N)] \\
- *  =& 1/(2N) [-D[g_{jk}N^jN^k - N^2] - N^jN^k D[g_{jk}]
+ *  \frac{1}{2} N^2 t^a t^b \Pi_{ab} - \frac{1}{2} N N^i t^a t^b \Phi_{iab}
+ *  =& \frac{1}{2} N^2 t^a t^b t^c \partial_c \psi_{ab}
+ *       - \frac{1}{2} N N^i (-(2/N) \partial_i N) \\
+ *  =& \frac{1}{2} N^2 [-(1/N^3) D[g_{jk} N^j N^k - N^2] \\
+ *           - (N^j N^k / N^3)D[g_{jk}] \\
```

Comment:
I think you're missing a `&` before the `-`. i.e.
```cpp
 *           &- (N^j N^k / N^3)D[g_{jk}] \\
```
same with the next line. Just a small typo :)

Reply:
fixed